### DISCUSSION TITLE
0.45 to be released when?

### DISCUSSION THREAD

**Discussion by flygfantast** [jvde-github/AIS-catcher#95] [General]
Just wondering when the 0.45 will be released? I'm interested int the Postgre support. :)

Or if it's possible to make a new Windows build.

Best regards
Niklas

**Comment by jvde-github** [comment]
Hi Niklas,

The latest builds can be found here: 

https://github.com/jvde-github/AIS-catcher/tree/decb1499a7b4f524adb9cf89b182110492ca6048/Edge

Curious how this works for you. Feedback welcome.

Cheers, Jasper

**Reply by flygfantast** [reply]
I'm struggling to get the connection to the DB. What parameters are to be used in the -D switch=

Now I'm getting a message that "no password supplied"

**Comment by jvde-github** [comment]
This is what worked for me (pushing UDP received NMEA in a database). 
```
.\AIS-catcher.exe -x 192.168.1.239 4002 -D postgresql://postgres:secret@localhost/ais
```

`ais` is the database and `secret` the password used for the database.

**Reply by flygfantast** [reply]
Thanks! That did the trick! Now the database is being populated with data :-)
